dev/NavigationView/NavigationView_InteractionTests/CommonTests.cs

```diff
@@ -1459,7 +1459,7 @@ public void EnsureClearingListIsSafe()
         [TestProperty("Description", "Ensure that the NavigationView button is rendering as expected if it's targeting RS3")]
         public void VerifyShouldPreserveNavigationViewRS3Behavior()
         {
-            using (var setup = new TestSetupHelper(new[] { "NavigationView Tests", "NavigationView PreserveRS3 Test" }))
+            using (var setup = new TestSetupHelper(testNames: new[] { "NavigationView Tests", "NavigationView PreserveRS3 Test" }, shouldRestrictInnerFrameSize: false))
```

Comment:
Can you add a comment saying why this should be set to false?

Reply:
done